### ISSUE TITLE
"Move All Products To Cart" and "Delete All" link is not working in Wishlist grid.

### ISSUE THREAD

**Issue by Jyoti-Singh1** [bagisto/bagisto#144]
Framework Version: 0.1.1

Product does not move to cart when click on "Move All Products To Cart". And its also not getting deleted when click on "Delete All".

**Comment by prashant-webkul** [comment]
Move all product to cart had been removed due to case of configurable product in the wishlist.
Since the flow of wishlist is that we can add the simple products in the wishlist or configurable only
but we cannot select variants to add in the wishlist.

Assume a case when there are some simple products and some configurable products added in the wishlist and we used move all to cart then only simple products would be really moved to cart but for all the configurable products the customer had to select options leading to a variant of that configurable product.

Since the original action on individual wishlist item for moving it to cart leads to item added to cart in case of simple product and select option for the configurable product followed by removal of that item added in the wishlist.

So, the incomplete experience was not fit to be involved into the system and hence the move all items to cart functionality has been removed.
